cli/src/test/java/org/jboss/gm/cli/MainTest.java

```diff
@@ -211,6 +215,8 @@ public void testInvokeAlignment() throws Exception {
 
     @Test
     public void testInvokeAlignmentFails() throws Exception {
+        assumeTrue(!SystemUtils.IS_OS_WINDOWS);
```

Comment:
We've got a right mix of asserting platforms ... I think we need to discuss whether - we need this, whether it can be simplified etc.

Reply:
I am removing the assertions from this file.